### PR TITLE
add check for device with cap 7.x

### PR DESCRIPTION
**Pull request by grimoire**
- paged attention kernel does not work on triton[2.2.0~2.3.1] on device with cap7.x.
- vicuna does not have `mlp_bias`.
- 7.x device doesn't t support bf16.

### CODE REVIEW

File: lmdeploy/pytorch/check_env/__init__.py [hunk 1]
```diff
@@ -91,6 +91,20 @@ def check_env_triton():
...
+        TRITON_VER_220 = version.parse('2.2.0')
+        TRITON_VER_231 = version.parse('2.3.1')
+
+        if device_cap[0] <= 7:
+            if (triton_version >= TRITON_VER_220
+                    and triton_version <= TRITON_VER_231):
+                err = RuntimeError(
+                    'Attention triton kernel does not fully support '
+                    'triton[2.2.0~2.3.1] on device with capability<8. '
+                    'Please change your triton version.')
```
Comment (lvhan028) [inline lmdeploy/pytorch/check_env/__init__.py:105]: better to recommend a triton version in the error message

Comment (lvhan028) [reply lmdeploy/pytorch/check_env/__init__.py:105]: So, on T4 or V100, triton 3.0 is suggested?

**Review by RunningLeon — APPROVED** [review]
LGTM

File: lmdeploy/pytorch/check_env/__init__.py (hunk 1 above)
Comment (grimoire) [reply lmdeploy/pytorch/check_env/__init__.py:105]: both 2.1.0 and 3.0 works on llama, 3.0 has not been sufficiently tested.

Comment (lvhan028) [reply lmdeploy/pytorch/check_env/__init__.py:105]: "Please change your triton version.", so , which one?

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**